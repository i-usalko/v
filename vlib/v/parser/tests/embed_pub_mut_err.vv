struct Context {}

struct App {
pub mut:
	Context
}

fn main() {
}
