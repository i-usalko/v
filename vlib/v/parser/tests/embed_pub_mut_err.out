vlib/v/parser/tests/embed_pub_mut_err.vv:6:1: error: expecting type declaration
    4 | pub mut:
    5 |     Context
    6 | }
      | ^
    7 |
    8 | fn main() {